tests/testthat/test-Zoopsynther.R

```diff
@@ -159,10 +159,9 @@ Data_filtered<-data_com_filtered%>%
             by="SampleID")
 
 test_that("Community approach does not change overall CPUE", {
-  expect_equal(Data_filtered$CPUE_base, Data_filtered$CPUE_com)
+  expect_equal(round(Data_filtered$CPUE_base, 6), round(Data_filtered$CPUE_com, 6))
```

Comment:
Now that you found the issue with this, did you try it again without the rounding and see if it passes? If it can pass without the rounding my preference would be to remove this change.

Reply:
YES! Test passed without rounding. I'll remove that.